### PR TITLE
EVG-16721: Replace usage of query.patchTasks with version.tasks

### PR DESCRIPTION
**Pull request by minnakt**
[EVG-16721](https://jira.mongodb.org/browse/EVG-16721)

### Description 
This PR removes the usage of `patchTasks` since `patchTasks` will be deprecated in favor of `version.tasks`.

### Screenshots
- No visible changes

### Testing 
- Existing tests should pass once corresponding PR is merged (set-module patch [here](https://spruce.mongodb.com/version/62e1716c0305b9019b51097c/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC))

### Evergreen PR 
- evergreen-ci/evergreen/pull/5823

### CODE REVIEW

File: src/gql/generated/types.ts [hunk 1]
```diff
@@ -2134,6 +2136,23 @@ export type VersionTaskStatusStatsArgs = {
   options?: InputMaybe<BuildVariantOptions>;
 };
 
+/** Version models a commit within a project. */
+export type VersionVersionTasksArgs = {
```
Comment (khelif96) [inline src/gql/generated/types.ts:2140]: supports my argument for using version.Tasks instead. This would become `VersionTasksArgs` instead.

**Review by khelif96 — APPROVED** [review]
LGTM with the exception of the comment on the other PR

**Comment by minnakt** [comment]
evergreen retry

**Comment by minnakt** [comment]
Seems to have fixed itself after someone else committed to main and I merged that commit into my branch 🤠

**Pull request merged**